### ISSUE TITLE
Please add accessibility support for screen readers

### ISSUE THREAD

**Issue by balajip8811** [a85/POSTMan-Chrome-Extension#13679]
### Is there an existing request for this feature?

- [x] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

Hi,
I use NVDA screen reader, lot of buttons are unlabeled and proper focus is not maintained when a dialogue box is opened. no labels are wrapped for radio buttons.
even though i am blind i can use vscode and work as team lead but facing  issues in postman .
can u please fix these please.
please try to follow wcag guidelines and do proper testing for the screen reader. please  include this in ur  dev cycle. thanks.

### Describe the solution you'd like

i want  all the functionality should be usable by screen readers. 
 try to implement key board  shortcuts  for functionality.
please  put  requests in collection in a table so that it is easy to navigate with screen reader.
please use a11y components.
